addons/fix-pasted-scripts/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Do not automatically run duplicated blocks",
-  "description": "Fixes a bug in the Scratch editor, where duplicating a block and attaching it to a script causes the script to run.",
+  "name": "Not-automatically-ran duplicated blocks",
```

Comment:
This is not German. Do not hyphenate words, it's okay not to nounify in this case.